src/ClusterManager/job.py

```diff
@@ -219,6 +234,35 @@ def _get_cluster_config(self, key):
             return self.cluster[key]
         return None
 
+    #only add blobfuse since we only support one plugin right now
+    def get_plugins(self):
+        '''
+        Plugin example:
+            "blobfuse": {
+                "enabled":true,
+                "name":"blobfuse",
+                "accountName":"YWRtaW4=",
+                "accountKey":"MWYyZDFlMmU2N2Rm",
+                "containerName":"blobContainer",
+                "mountPath":"/data",
+                "secreds":"bb9cd821-711c-40fd-bb6f-e5dbc1b772a7"
+                }
+        '''
+        if "plugins" not in self.params.keys():
+            return
+        plugins = self.params["plugins"]
+
+        ret = {}
+        if plugins["blobfuse"] is not None:
```

Comment:
`"blobfuse"` may not exist in `plugins`.

Reply:
yeah, modify it